index.html

```diff
@@ -1,63 +1,150 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Sajjda Portfolio</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
 
-  <body>
-    <header id="main-header">
-      <div class="toolbarContainer">
-        <h2 id="logo">Sajjad</h2>
-        <nav>
-          <img src="images/menu-icon.png" id="menu-icon" alt="menu icon" />
-        </nav>
-      </div>
-    </header>
-    <section id="mainSection">
-      <article>
-        <section class="Introduction">
-          <div class="IntroContainer">
-            <div id="primary-text">I’m Sajjad Glad to see you!</div>
-            <div id="secondary-text">
-              I’m a software developer! I can help you build a product , feature
-              or website Look through some of my work and experience! If you
-              like what you see and have a project you need coded, don’t
-              hestiate to contact me.
-            </div>
-            <div id="connect">LET'S CONNECT</div>
-            <div id="socialmedia">
-              <a href="https://twitter.com/sajjadrajaee">
-                <img src="images/social/twitter.svg" alt="social icons" />
-              </a>
-              <a href="https://linkedin.com/in/sajjadrajaee">
-                <img src="images/social/Linkedin.svg" alt="social icons" />
-              </a>
-              <a href="#">
-                <img src="images/social/M.svg" alt="social icons" />
-              </a>
-              <a href="https://github.com/sajjadrajaee">
-                <img src="images/social/Github.svg" alt="social icons" />
-              </a>
-              <a href="#">
-                <img src="images/social/Vector.svg" alt="social icons" />
-              </a>
-            </div>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Sajjad Portfolio</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header>
+    <h2 id="logo">Sajjad</h2>
+    <nav id="mobile-nav">
+      <img src="images/menu-icon.png" id="menu-icon" alt="menu icon" />
+    </nav>
+    <nav id="desktop-nav">
+      <ul>
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
+      </ul>
+    </nav>
+  </header>
+  <section id="mainSection">
+    <article>
+      <section class="Introduction">
+        <div class="IntroContainer">
+          <div id="primary-text">I’m Sajjad Glad<br /> to see you!</div>
```

Comment:
- [ ] I think you should use the `h1` tag instead of the `div` tag here.  The main title of the page should be in a `h1` tag. Please fix it. :+1: